src/Skia/Avalonia.Skia/WriteableBitmapImpl.cs

```diff
@@ -62,12 +67,39 @@ public WriteableBitmapImpl(PixelSize size, Vector dpi, PixelFormat? format = nul
         public void Draw(DrawingContextImpl context, SKRect sourceRect, SKRect destRect, SKPaint paint)
```

Comment:
This change only makes sense when we could just update a dirty region in the buffer. Without that we gain nothing here and just hold an additional copy. What am I missing here?

Reply:
The idea was that you copy the writeable bitmap to a GPU surface to speed up rendering. Of course, if you update the bitmap every frame there's little to be gained. But for scenarios where you don't update every frame, performance will be better.

Reply:
I think we can change WritableBitmap to always use a SkImage under the hood and use it for software and hardware rendering. The frame buffer will be just a region in memory that is wrapped by that SKImage instance.

https://docs.microsoft.com/de-de/dotnet/api/skiasharp.skimage.frompixels?view=skiasharp-1.60.3#SkiaSharp_SKImage_FromPixels_SkiaSharp_SKImageInfo_System_IntPtr_